Tests/PHPUnit_Framework_TestCase.php

```diff
@@ -1,34 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sonata Project package.
- *
- * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\NewsBundle\Tests;
-
-/**
- * NEXT_MAJOR: Remove this class when dropping support for < PHPUnit 5.4.
```

Comment:
We drop PHPUnit 4, or is the PR title wrong?

Reply:
Both things are true, but I think we will never use anything less than whatever the last 5 version is, so this is fine.